src/PlatformHelpers.web.js

```diff
@@ -1,3 +1,8 @@
-import { BackHandler } from 'react-native';
+import React from 'react';
+import { BackHandler, Text } from 'react-native';
 
-export { BackHandler };
+const MaskedViewIOS = () => {
+  <Text>MaskedViewIOS not available on Reacht Native Web</Text>;
```

Comment:
Is this the correct fallback behavior? Shouldn’t we just let masking silently fail, so that the stack header doesn’t get messed up with this text?

Reply:
Fixed.